src/WoodburyMatrices.jl

```diff
@@ -98,4 +98,36 @@ function A_ldiv_B!(W::Woodbury, B::AbstractVector)
     B
 end
 
+"""
+Build `rowspec`, `valspec`, `colspec` such that the product
+
+`out = rowspec * valspec * colspec` will be equivalent to:
+
+```julia
+out = zeros(n, n)
+
+for (i, j, v) in args
+    out[i, j] = v
+end
+```
+
+"""
+function _build_woodbury_specs{T}(::Type{T}, n::Int, args::Tuple{Int, Int, Any}...)
```

Comment:
The code is fine---it's not high-performance, but this function doesn't need to be, if we don't export it, because you're using it only for small problems.

The help text is a little hard to follow: what is `args`? (it's clear if you read the code, but...). You might want to mention that `rowspec`, `valspec`, and `colspec` are all sparse matrices.